.github/workflows/tests_primary.yml

```
@@ -119,7 +119,34 @@ jobs:
     - run: npm ci
       env:
         DEBUG: pw:install
+        PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1
     - run: npm run build
+    # Debugging for https://github.com/microsoft/playwright/issues/17394
+    - name: Download Playwright using Node.js debug build
+      if: matrix.os == 'macos-latest' || matrix.os == 'ubuntu-latest'
+      run: |
+        base_filename="node-linux-dbg-x64.tar.gz"
+        if [ "${{ matrix.os }}" == "macos-latest" ]; then
```

nit: never use single `[` in bash, always use `[[` and `]]`

```suggestion
        if [[ "${{ matrix.os }}" == "macos-latest" ]]; then
```